src/sentry/api/fields/avatar.py

```diff
@@ -62,3 +69,35 @@ def is_valid_size(self, width, height):
         if width > self.max_dimension:
             return False
         return True
+
+
+class SentryAppLogoField(AvatarField):
+    def __init__(
+        self,
+        max_size=settings.SENTRY_MAX_AVATAR_SIZE,
+        min_dimension=MIN_SVG_DIMENSION,
+        max_dimension=MAX_SVG_DIMENSION,
+        **kwargs,
+    ):
+        super().__init__(**kwargs)
+        self.max_size = max_size
+        self.min_dimension = min_dimension
+        self.max_dimension = max_dimension
+
+    def to_internal_value(self, data):
+        if not data:
+            return None
+        data = b64decode(data)
+        if len(data) > self.max_size:
+            raise ImageTooLarge()
+
+        if SVG_RE.match(data.decode("utf-8")) is not None:
+            svg = load_svg_file(BytesIO(data))
+            if svg is not None:
+                viewbox = svg.get("viewBox").split()
+                width, height = [int(dimension) for dimension in viewbox][-2:]
+                if not self.is_valid_size(width, height):
+                    raise serializers.ValidationError("Invalid image dimensions.")
```

Comment:
So after taking a second look at this I think this might be an issue. The viewbox on an SVG is an internal coordinate mapping and doesn't necessarily indicate the real size of the SVG. I think it inherits its relative size from the parent component in the DOM, so if `viewBox="0 0 10 10"`, but the parent isn't constrained, the SVG can still take up the entire webpage. 

We should probably instead take a look at ensuring there are no `height` and `width` attributes specified on the `<svg>` tag, since those would translate to actual dimensions when we render.